scripts/install-web.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable unicorn/no-process-exit */
+const path = require('path');
+const { spawn } = require('child_process');
+const { existsSync } = require('fs');
+
+const projectRoot = path.resolve(__dirname, '../');
+const webExamplePath = path.resolve(projectRoot, './examples/react-web');
+const modulesPath = path.resolve(webExamplePath, './node_modules');
+
+const hasModules = existsSync(modulesPath);
+
+if (!hasModules) {
+  installDependencies();
+} else {
+  process.exit(0);
+}
+
+function installDependencies() {
+  console.log('Installing web dependencies...');
+  const npmInstall = spawn('npm', ['install'], { cwd: webExamplePath });
+
+  npmInstall.stdout.on('data', function(data) {
+    console.log(data.toString());
+  });
+
+  npmInstall.stderr.on('data', function(data) {
+    console.error(data.toString());
+  });
```

Comment:
The `npm install` subprocess seems to be silent for me (i.e. I can't see any output from it so it just appears that the terminal is hanging while it is doing the install).  Perhaps we're not subscribing to the data stream correctly?

Reply:
I’m not 100% sure of best practices, but if the aim is just to let `npm install` run and display it’s output, I think the simplest is using the `stdio` option:

```js
const npmInstall = spawn(
  'npm',
  ['install'],
  {
    cwd: webExamplePath,
    stdio: 'inherit'
  }
); 
```

This outputs everything straight to the parent `stdout`.

Reply:
Node docs: https://nodejs.org/dist/latest-v12.x/docs/api/child_process.html#child_process_options_stdio

Reply:
Thanks @delucis , using `inherit` solved the issue!